frontend/app/src/components/defi/Overview.vue

```diff
@@ -120,8 +129,18 @@ export default class Overview extends Vue {
   }
 
   get icon(): string {
+    // until we can check whether a file exists locally, let's make a whitelist of protocols where we prefer our own icon
+    const protocolWhitelist: string[] = ['aave', 'makerdao', 'compound'];
+
+    const isWhitelisted: boolean =
+      protocolWhitelist.findIndex(
+        protocol => protocol == this.summary.protocol.name.toLowerCase()
```

Comment:
Would `protocolWhitelist.includes(this.summary.protocol.name.toLowerCase())` work here instead of find index?

Reply:
Yes good call, I was trying to remember what that method was!

Reply:
No worries, it is almost impossible to remember everything.